salt/_modules/metalk8s_monitoring.py

```diff
@@ -0,0 +1,173 @@
+"""Utiliy methods to interact with MetalK8s monitoring.
+"""
+
+from salt.exceptions import CommandExecutionError
+
+from datetime import datetime, timedelta
+
+MISSING_DEPS = []
+
+try:
+    import requests
+    from requests.packages.urllib3.exceptions import InsecureRequestWarning
+except ImportError:
+    MISSING_DEPS.append('requests')
+
+__virtualname__ = 'metalk8s_monitoring'
+
+CONTROL_PLANE_INGRESS_PORT = 8443
+
+
+def __virtual__():
+    if MISSING_DEPS:
+        error_msg = 'Missing dependencies: {}'.format(', '.join(MISSING_DEPS))
+        return False, error_msg
+
+    requests.packages.urllib3.disable_warnings(InsecureRequestWarning)
```

Comment:
I'd rather make sure we trust the CA than go out of our way to ignore warnings globally (what about other consumers of this module?). It's the Ingress CA, so you can get the bundle at `/etc/metalk8s/pki/nginx-ingress/ca.[crt|key]`.

Reply:
Well, that's only if we use the Ingress. But we're inside the cluster, so we shouldn't.

Reply:
Removed